suisa_sendemeldung/suisa_sendemeldung.py

```diff
@@ -678,6 +678,7 @@ def get_xlsx(data: Any, args: ArgparseNamespace) -> BytesIO:  # noqa: ANN401
 
     xlsx = BytesIO()
     workbook: Workbook = Workbook()
+    workbook.iso_dates = True
```

Comment:
I want to compare this with the template provided by SUISA before we commit this change

using ISO dates feel natural and like the right thing to do, but i'm not sure if their template also uses ISO dates. my current understanding is that ISO dates are defined to be the norm in ECMA-376 and have been so since OOXML